dashboard/app/views/test_execs/_show_valid.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= content_tag 'div', id: 'test-exec-info', data: {test_exec: @test_exec} do %>
+  <p>
+    UUID: <span id="test-exec-uuid"><%== @test_exec.uuid %></span>
+  </p>
+  <p>
+    Status: <span id="test-exec-status"><%== @test_exec.status %></span>
+  </p>
+  <p>
+    Counters:
+    <span id="test-exec-counters">
+      <% if @test_exec.counters.has_key? 'total' %>
+        <%== '.' * @test_exec.counters['total'] %>
+      <% end %>
+    </span>
+  </p>
+<% end %>
+
+<script type="text/javascript">
+  var testExecUUID = $('#test-exec-uuid').html();
```

Comment:
@mDibyo please wrap this in a closure so we avoid global variables. (preferable a SEAF - http://markdalgleish.com/2011/03/self-executing-anonymous-functions/)